application/src/main/resources/application.yml

```diff
@@ -16,6 +16,7 @@
 
 server:
   port: 8080
+  # nodename: 'local.test.org'
```

Comment:
this property should be in the local and docker profiles since it is something which should always be changed